packages/core/lib/modules/data-migration.test.js

```diff
@@ -1,27 +1,54 @@
 import {Wallet} from './wallet';
+import walletLegacyData from '../test/fixtures/legacy-wallet-schema.json';
 import {mockDockService} from '../services/test-utils';
-import walletLegacyData from '../test/fixtures/lagacy-wallet-schema.json';
 
 describe('DataMigration', () => {
   let unmockDockService;
-  let wallet: Wallet;
 
   beforeAll(async () => {
     unmockDockService = await mockDockService();
-    global.localStorage.setItem(
-      'dock-wallet',
-      JSON.stringify(walletLegacyData),
-    );
-    wallet = await Wallet.create();
-    await wallet.ensureNetwork();
   });
 
-  it('expect to create wallet with legacy data', async () => {
+  describe('migrate wallet from v0.1 to v0.2', () => {
+    let wallet: Wallet;
+
+    beforeAll(async () => {
+      global.localStorage.setItem(
+        'dock-wallet',
+        JSON.stringify(walletLegacyData),
+      );
+      wallet = await Wallet.create();
+
+      await wallet.load();
+      await wallet.ensureNetwork();
+    });
+
+    it('expect to migrate accounts', async () => {
+      const accounts = await wallet.accounts.getAccounts();
+      expect(accounts.length).toBe(2);
```

Comment:
how do we know the accounts were migrated?

Reply:
The .getAccounts() will return 0 items in case the migration didn't run. But i`ve also added another check ` expect(wallet.migrated).toBeTruthy();` so that can be used for future migrations too

Reply:
No, the new getAccounts() will be using an `Address` document and resolve the related documents such as `Currency`, `Keypair` and `Mnemonic`.  The old schema was using an `Account` document which is not valid for universal wallet, this document type will not be used anymore

Reply:
The getAccounts() here is used to make sure the wallet has the accounts we expected, and `wallet.migrated` will assert the migration was performed

Reply:
then we should cover all the scenarios